.eslintrc.json

```diff
@@ -27,6 +27,12 @@
     ],
     "rules": {
         "no-console": "off",
+        "@typescript-eslint/array-type": [
+            "error",
+            {
+                "default": "array"
```

Comment:
Changes this so that we could define arrays like `const x: string[] = ['a', 'b'];`